src/mvt_getter.py

```diff
@@ -51,17 +51,36 @@ def get_vectors(minzoom, maxzoom, x, y, style, vec):
                     add = "(" + add + ")"
                     adp.append(add)
 
+    if "name" in column_map and locale == "en":
```

Comment:
this should go into name:en tag instead of name. the tiles should be style-agnostic, although can be augmented. either generate all languages you have quirks for, or configure a list of the ones you need.

on the rendering side you then just coalesce(name:en, name).

Reply:
I made a change. I prefer the way you described but there still might be confusing things like:
`eval(any(tag("name:en", "int_name", "name")))` and `coalesce("name:en", "int_name", "name") as "name:en"` in sql would result in unused `int_name` and `name` in mvt.

Reply:
yeah. `eval(any(tag("name:en", tag("int_name"), tag("name"))))` is not an expected usage: designer should write just `text: name`. Locale is not a task of style designer.

Basically, there are some roles that should be kept rather distinct:

 - designer: writes mapcss, chooses colors and fonts, expects things to magically work [mapcss]
 - mapper: fills in data to osm.org so there are no blank spots on the map [osm.org]
 - engineer: makes the magic expected by designer work (labels priorities, generalization, locales, swapping out renderer for other one, ...) [kothic+js+db internals]